File: src/multiquines.pl
my $S=q:b"DATA";my @L=$S.split("\t");my $I=@L.grep-index(@*ARGS[0]);print @L[$I[0]+1]~$S.trans(["\t","\n",'"',"\\"]=>['\t','\n','\\"',"\\\\"])~@L[$I[0]+2]

File: src/multiquines.go
package main;import(."fmt";."os";."strconv";."strings");func main(){S:="DATA";L:=Split(S,"\t");for I,s:=range L{if s==Args[1]{S=Quote(S);Print(L[I+1]+S[1:len(S)-1]+L[I+2]);}}}
